include/sentry.h

```diff
@@ -23,13 +23,7 @@ extern "C" {
 #endif
 
 /* SDK Version */
-#ifdef __ANDROID__
-#    define SENTRY_SDK_NAME "sentry.native.android"
-#else
-#    define SENTRY_SDK_NAME "sentry.native"
-#endif
 #define SENTRY_SDK_VERSION "0.4.15"
-#define SENTRY_SDK_USER_AGENT SENTRY_SDK_NAME "/" SENTRY_SDK_VERSION
```

Comment:
We should keep these definitions in the header file. IMO it would be a breaking change if we just remove those.
I would argue to just wrap them in `#ifndef` and then the definition from CMake/outside will take precedency.

Reply:
Yes, thats what I meant by "wrap them in `#ifndef`" ;-)

It should not be possible to override the version.

Reply:
changed as discussed